config/app.php

```diff
@@ -84,7 +84,7 @@
     |
     */
 
-    'locale' => 'ro',
+    'locale' => env('APP_LANG', 'en'),
```

Comment:
nu e nevoie să setăm limba din env

```suggestion
    'locale' => 'ro',
```

Reply:
eu zic ca da , nu stiu sa se voteze